collectors/ebpf.plugin/ebpf.c

```diff
@@ -740,15 +751,17 @@ void ebpf_print_help()
             " --process or -p     Enable charts related to process run time.\n"
             "\n"
             " --return or -r      Run the collector in return mode.\n"
-            "\n",
+            "\n"
+            " --shm or -b         Enable chart related to shared memory tracking.\n"
```

Comment:
Just curious, how do you choose which single letter to use? 😄

Reply:
When we had less threads the selection was easy, the first letter of the thread, nowadays it depends of each situation. For this, for example, @UmanShahzad can answer. :laughing:

Reply:
I tried to make it somewhat relevant to the long flag for the previous charts I did, but now we've run out of relevant letters. So I've just started from `a` and will pick the first letter not available. :smile: 

Since we want to introduce a huge number of more collectors next quarter, I think we'll run out of alphabetical space and just have to ignore the short option.

(And as an aside, I tried using the long form to only run the `shm` thread, and it still runs everything, so there's a logic issue somewhere).

Reply:
Thanks for the answers!

> but we should solve it at some point

Consider creating an issue so it won't get forgotten.

Reply:
Made https://github.com/netdata/netdata/issues/11568